src/store/inflight_activation.rs

```diff
@@ -131,6 +186,11 @@ impl InflightActivation {
                 })
         })
     }
+
+    /// A convenience function to get the TaskActivation
+    pub fn get_activation(&self) -> TaskActivation {
+        self.activation.clone().unwrap()
```

Comment:
This seems like a risky method to call outside of tests. Should we mention that in the doc string?